cmd/libsnap-confine-private/cgroup-freezer-support.c

```diff
@@ -119,7 +119,7 @@ bool sc_cgroup_freezer_occupied(const char *snap_name)
 		debug("found process %s belonging to user %d",
 		      line_buf, statbuf.st_uid);
 		return true;
-	} while (num_read > 0);
```

Comment:
This is just a cleanup since the `while (num_read > 0);` will never evaluate to false since at line 101 we break on `num_read <= 0`.